gems/smithy/lib/smithy/views/client/types_rbs.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+module Smithy
+  module Views
+    module Client
+      # @api private
+      class TypesRbs < View
+        def initialize(plan)
+          @plan = plan
+          @model = plan.model
+          super()
+        end
+
+        def namespace
+          Util::Namespace.namespace_from_gem_name(@plan.options[:gem_name])
+        end
+
+        def types
+          @types ||= Model::ServiceIndex
+                     .new(@model)
+                     .shapes_for(@plan.service)
+                     .select { |_key, shape| %w[structure union].include?(shape['type']) }
+                     .map { |id, structure| Type.new(id, structure, @model) }
+        end
+
+        # @api private
+        class Type
+          def initialize(id, structure, model)
```

Comment:
Super ridiculous nit and feel free to ignore but put model first (convention with other classes)

Reply:
yea, good call.  Fixed.